db/re/item_db.txt

```diff
@@ -10143,12 +10163,19 @@
 19223,Illusion_Cap,Illusion Cap,4,0,,400,,15,,1,0x000654E3,63,2,256,,120,1,14,{ bonus2 bAddClass,Class_All,1; bonus bBaseAtk,(2*getrefine()); },{},{}
 19247,Illusion_Fancy_Flower,Illusion Fancy Flower,4,0,,100,,0,,1,0xFFFFFFFF,63,2,256,,120,0,4,{ bonus bMatkRate,1; bonus bMatk,(2*getrefine()); },{},{}
 //
+19241,C_Magical_Booster_,Costume Magical Booster,4,,,30,,10,,1,0xFFFFFFFF,63,2,2048,0,,,873,{},{},{}
+19242,C_Neko_Headress,Costume Neko Headress,4,,,10,,20,,1,0xFFFFFFFF,63,2,1024,0,,,816,{},{},{}
```

Comment:
according to https://www.divine-pride.net/database/item/19242

19242,Kitten_Headdress,Kitten Headdress,4,,,100,,20,,1,0xFFFFFFFF,63,2,1024,0,,,816,{ .@r = getrefine(); bonus bAllStats,1; bonus bAspdRate,5; if (.@r>=9) bonus bVariableCastrate,-10; if (.@r>=12) bonus bNoCastCancel; if (Baselevel < 100) bonus2 bExpAddRace,RC_All,10; else bonus2 bExpAddRace,RC_All,4; },{},{}


and need to add combo with Blush
19242:5040,{ bonus bHealPower,5; }